deploy/charts/harvester/dependency_charts/whereabouts/templates/daemonset.yaml

```diff
@@ -1,3 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: whereabouts-config
+  namespace: kube-system
+  annotations:
+    kubernetes.io/description: |
+      Configmap containing user customizable cronjob schedule
+data:
+  cron-expression: "30 4 * * *" # Default schedule is once per day at 4:30am. Users may configure this value to their liking.
```

Comment:
Could we separate the ConfigMap manifest from daemonset.yaml into a new configmap.yaml? Though there's no difference, I think it's more well-organized in terms of structure. Thank you.